Makefile

```
@@ -50,9 +50,9 @@ run: nina
 	@. $(activate) && \
 	python -m nina
 
-.make.deps: requirements-dev.txt $(venv) $(venv)/bin/nina
+.make.deps: setup.cfg $(venv) $(venv)/bin/nina
 	@. $(activate) && \
-	pip install -r requirements-dev.txt && \
+	pip install -e .[dev,testing] && \
```

Be sure to add quotes around [ ].
I was having some issues with ZSH and that syntax.
```suggestion
	pip install -e '.[dev,testing]' && \
```

---

Not sure if `make` uses your default shell, but instead use `sh` or something.

---

This worked fine in emacs, eshell and zsh for me. I seems to work while being quote too, so I'll apply to be sure.